### PR TITLE
Updating ReadMe for chatroom instructions

### PR DESCRIPTION
**Pull request by jusasin**
### Issue Number 
Continues work from issue #5 

### Description
Add instructions to the main readme in the root directory on how to run the chat room. Includes directions on what to do and what commands to run. To confirm that it works, a description of what the user should see is included. No AI was used to generate the instructions. 

### Testing
No test are needed since it is just a readme file. 

### Possible Errors
None

### CODE REVIEW

**Comment by blake-gilmore** [comment]
Much needed instructions. Now I won't have to search our discord messages to know how to run the server haha

**Review by blake-gilmore — APPROVED** [review]

**Review by tbg510 — APPROVED** [review]
Nice to always have more documentation that we can find in one place. Let's keep it up !

File: README.md [hunk 1]
```diff
@@ -28,4 +28,16 @@ This project utilizes the Jest testing suite for its unit tests. Our goal of tes
...
+    3.3 'npm run test:coverage' - Jest will be ran like usual but output a comprehensive report of the coverage for the app. It lists the files, percentages of statements, branches, functions, and lines, and lists the uncovered lines in the files. It will also print the targeted testing coverage percent and whether or not we've reached it.  
+
+## How to run the Chatroom
+This project utilizes socket.io to create a connection between the users and the chatroom. In order to run the chatroom on your own machine you will need to follow the following steps:
+
+1. Open two terminal windows
+2. In the first terminal - Navigate to the small-talk directory:  
+    2.1 'npm install' - used to install any missing dependencies  
+    2.2 'npm run dev' - This will simply run the website on your localhost 
+3. In the second terminal - Navigate to the chatroom-socket-server directory:
```
Comment (aiharc1) [inline README.md:40]: Thank you for adding the socket side instructions. I noticed we didn't have that documented as well.

**Review by aiharc1 — APPROVED** [review]
Nice work Justin. The addition of server side instructions were much needed.

**Pull request merged**